scico/examples.py

```diff
@@ -317,3 +317,52 @@ def spnoise(img: Array, nfrac: float, nmin: float = 0.0, nmax: float = 1.0) -> A
         imgn = imgn.at[spm < nfrac - 1.0].set(nmin)
         imgn = imgn.at[spm > 1.0 - nfrac].set(nmax)
     return imgn
+
+
+def create_3D_phantom_foam(
```

Comment:
I suggest `create_3D_foam_phantom`.

Reply:
Also, better to move it to be directly after the other phantom construction functions.

Reply:
Is this an extension of the `xdesign` foam construction code? If so, mention the corresponding `xdesign` function in the docs

Reply:
Made the changes. It tries to do something like what xdesign does in 3D but it doesn't use any of xdesign code and most likely differs from their approach considerably. This is based on what Mike sent in mattermost (his name should be added during the PR merge).

Reply:
OK, then no need for the `xdesign` reference.